examples/multiedgetracking/detection/estimator/__init__.py

```diff
@@ -0,0 +1,16 @@
+import importlib
```

Comment:
Missing `kubeedge` file header comment

Reply:
Fixed. There are multiple files where the copyright header is missing, but we can probably do this later.